packages/nimble-components/src/utilities/tests/timeout.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Timeout for use in async tets.
+ */
+export async function timeout(ms = 0): Promise<void> {
```

Comment:
Since this is being used in a pageobject and pageobjects are in the public API, it cannot be in a tests folder since those are removed from build. This file should be moved to the `src/utilities/testing` folder:
https://github.com/ni/nimble/tree/main/packages/nimble-components/src/utilities/testing

Reply:
Actually maybe just move is to this file? It seems to be a catch all file and already has some timing functions init: https://github.com/ni/nimble/blob/main/packages/nimble-components/src/utilities/testing/component.ts#L68

Reply:
to align with the other naming conventions (waitForEvent, and waitAnimationFrame) lets also change it to `waitTimeout`